tunedev/Banka
# #165482473 authorization setup

#### What does this PR do?
This pr sets up authorization logic on the app.

#### Description of Task to be completed?
- protects routes according to user type

#### How should this be manually tested?
- Clone this repo and cd into its root directory
- Run `npm install` and `npm run devstart`
- git checkout to this branch
- Make a request to any routes asides sign-in and sign-up routes
- An error message requesting for a token should be gotten as a response

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#165482473](https://www.pivotaltracker.com/story/show/165482473)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
